### PR TITLE
Bump Parity 3.1.0 -> 3.2.0

### PR DESCRIPTION
**Pull request by geoffharcourt**
```
❯ shasum ~/Downloads/parity-3.2.0.tar.gz
18f75daf1878e4c6ef06dd60b6752223b9f9106f  /Users/geoff/Downloads/parity-3.2.0.tar.gz
```

### CODE REVIEW

File: Formula/parity.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@
 class Parity < Formula
   homepage "https://github.com/thoughtbot/parity"
   head "https://github.com/thoughtbot/parity.git"
-  sha256 "c804bca89d99f8cfacde6098959a198b9ef347ad8ce40759133fd766206d7b04"
-  url "https://github.com/thoughtbot/parity/archive/3.1.0.tar.gz"
+  sha256 "18f75daf1878e4c6ef06dd60b6752223b9f9106f"
+  url "https://github.com/thoughtbot/parity/archive/3.2.0.tar.gz"
```
Comment (mike-burns) [inline Formula/parity.rb]: Missing a `v`.

```suggestion
  url "https://github.com/thoughtbot/parity/archive/v3.2.0.tar.gz"
```

File: Formula/parity.rb [hunk 2]
```diff
@@ -3,8 +3,8 @@
 class Parity < Formula
   homepage "https://github.com/thoughtbot/parity"
   head "https://github.com/thoughtbot/parity.git"
-  sha256 "c804bca89d99f8cfacde6098959a198b9ef347ad8ce40759133fd766206d7b04"
-  url "https://github.com/thoughtbot/parity/archive/3.1.0.tar.gz"
+  sha256 "18f75daf1878e4c6ef06dd60b6752223b9f9106f"
```
Comment (mike-burns) [inline Formula/parity.rb]: I get `fb4d72e7d84d7849c25589bf718580b46117f992f2b8b9d22a208bfad6d60a19` for the tarball from GH releases.

**Comment by mike-burns** [comment]
According to [this illegible man page for shasum(1) that I found online](https://ss64.com/osx/shasum.html), the default is SHA1. Maybe `shasum -a 256 /Users/geoff/Downloads/parity-3.2.0.tar.gz` will give the results we need.

**Comment by geoffharcourt** [comment]
@mike-burns it's been a while, and I was supposed to do 256. Thanks for spotting, I'll regenerate.

**Comment by mike-burns** [comment]
```
~% curl -LO https://github.com/thoughtbot/parity/archive/v3.2.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   125    0   125    0     0    646      0 --:--:-- --:--:-- --:--:--   647
100 10447    0 10447    0     0  25439      0 --:--:-- --:--:-- --:--:-- 25439
~% ck256 -v v3.2.0.tar.gz fb4d72e7d84d7849c25589bf718580b46117f992f2b8b9d22a208bfad6d60a19
yes
```

That and a `tar -ztf` look good. Thank you!

LGTM.

**Pull request merged**